src/libs/Navigation/AppNavigator/AuthScreens.js

```diff
@@ -71,7 +71,7 @@ const RootStack = createCustomModalStackNavigator();
 // When modal screen gets focused, update modal visibility in Onyx
```

Comment:
This comment is out of date now and we could add some more context about why we are waiting until the transition ends.

Separately, is there any reason to do this? `focus` and `transitionEnd` events should be reasonably the same when it comes to setting the "modal is visible" flag. Or did you see some reason why it doesn't work?

Reply:
Just to delay the Chain effect until the transition is over. `Focus` is fired earlier than the `transitionEnd` which trigger rerender for few components like `ReportActionCompose`.